src/components/ui/cart/cart.lineitem.jsx

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+import { login } from '../../../utils/AuthService.js';
+
+var Config = require('Config')
+
+class CartLineItem extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            quantity: this.props.item.quantity
+        }
+        this.handleQuantityChange = this.handleQuantityChange.bind(this);
+        this.handleRemoveBtnClicked = this.handleRemoveBtnClicked.bind(this);
+    }
+    handleQuantityChange(event) {
+        event.preventDefault();
+        let newQuantity = event.target.value;
```

Comment:
any reason you need to create this rather than just passing the value?

Reply:
yes, the event was being lost inside of the login()